timescaledb/how-to-guides/continuous-aggregates/continuous-aggregates-on-continuous-aggregates.md

```diff
@@ -0,0 +1,138 @@
+    daily continuous aggregate
+*   A real-time yearly continuous aggregate on the monthly continuous aggregate
+
+Queries on the hourly and daily continuous aggregates include real-time,
+non-materialized data from the source hypertable. Queries on the monthly
+continuous aggregate only return already-materialized data. Queries on the
+yearly continuous aggregate return materialized data from the yearly continuous
+aggregate itself, plus more recent data from the monthly continuous aggregate.
+However, the data is limited to what is already materialized in the monthly
+continuous aggregate, and doesn't get even more recent data from the source
+hypertable. This happens because the materialized-only continuous aggregate
+provides a stopping point, and the yearly continuous aggregate is unaware of any
+layers beyond that stopping point. This aligns with how stacked views work in
+PostgreSQL.
+
+To make queries on the yearly continuous aggregate access all recent data, you
+can either:
+
+*   Make the monthly continuous aggregate real-time
+*   Or redefine the yearly continuous aggregate on top of the daily continuous
+    aggregate
+
+## Roll up calculations
+
+When summarizing already-summarized data, be aware of how stacked calculations
+work. Not all calculations return the correct result if you stack them.
+
+For example, if you take the maximum of several sub-sets, then take the maximum
+of the maximums, you get the maximum of the entire set. But if you take the
+average of several sub-sets, then take the average of the averages, that is, in
+general, different from taking the average of all the data.
+
+To simplify such calculations when using continuous aggregates on top of
+continuous aggregates, you can use the [hyperfunctions][hyperfunctions] from
+TimescaleDB Toolkit, such as the [statistical aggregates][stats-aggs]. These
+hyperfunctions are designed with a two-step aggregation pattern which allows you
+to roll them up into larger buckets. The first step creates a summary aggregate
+that can be rolled up, just as a maximum can be rolled up. You can store this
+aggregate in your continuous aggregate. Then, you can call an accessor function
+as a second step when you query from your continuous aggregate. This accessor
+takes the stored data from the summary aggregate and returns the final result.
+
+## Restrictions
+
+There are some restrictions when creating a continuous aggregate on top of
+another continuous aggregate. In most cases, these restrictions are in place to
+ensure valid time-bucketing:
+
+*   You can only create a continuous aggregate on top of a finalized continuous
+    aggregate. This new finalized format is the default for all continuous
+    aggregates created since TimescaleDB 2.7. If you need to create a continuous
+    aggregate on top of a continuous aggregate in the old format, first [migrate
+    your continuous aggregate][migrate-cagg].
```

Comment:
```suggestion
    aggregates created since TimescaleDB 2.7. If you need to create a continuous
    aggregate on top of a continuous aggregate that is in the old format, you need to 
    [migrate your continuous aggregate][migrate-cagg] to the new format first.
```